src/patternfly/components/Page/page.scss

```diff
@@ -1,6 +1,20 @@
 // URL.com/guidelines#layout
 .pf-c-page {
   --pf-c-page--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-page--GridTemplateColumns: 1fr;
```

Comment:
It's probably hard to tell because the content isn't inside of the drawer, but I think it might look odd that the notification drawer has a width of 450px, and then at the md breakpoint, the drawer almost doubles in width. Was this intentional? Also what happens when the sidebar is visible again, does it display on top of the drawer?

<img width="800" alt="Screen Shot 2019-07-08 at 10 03 51 AM" src="https://user-images.githubusercontent.com/20118816/60816451-ca585200-a167-11e9-9a83-93b54baac68c.png">

Reply:
Yep, I went over the design with @kybaker and @mcarrano last week, and we decided to have it full width on mobile (like the wizard) and 450px on desktop. And yep, the sidebar should always appear on top of the drawer.

Reply:
Ok sounds good, I think it would be good to see an example of the sidebar opened on top of the drawer.

Reply:
I created another example so we show the collapsed and expanded states, so the react devs can see the classes needed to build for the two states. I think for how it looks with the sidebar open, we could point users to the react demo, since that would only be a visual demo and doesn't demo any code needed for a drawer state.What do you think?

We don't currently have any other page demos with the sidebar open, either. That said, I wonder if we should add 2 new demos to the page component to illustrate the expanded sidebar on mobile, and collapsed sidebar on desktop. We could always incorporate those states into the existing demos, too.

Reply:
I just added a comment underneath before I saw this one. But yeah I think it would be good to add new examples to the page component that show the sidebar opened and closed, I dont see an expanded state on it, so not sure what do to there.. but that sounds good then we can show what it looks like with both.